src/main/java/com/tstepnik/planner/domain/user/userregistrationdto/RegisterUserDto.java

```diff
@@ -0,0 +1,58 @@
+package com.tstepnik.planner.domain.user.userregistrationdto;
+
+import javax.validation.constraints.NotEmpty;
+
+public class RegisterUserDto {
+
+    @NotEmpty
```

Comment:
Besides not empty, you can also add really specific validators like `@Size` or even regex patterns for emails to check if email field value is actually an email. Same with password :)

Reply:
Could you give me example of regex which i can use here ?